everyvoice/demo/app.py

```diff
@@ -207,6 +208,11 @@ def create_demo_app(
         device
     )
     model.eval()
+
+    from everyvoice.base_cli.helpers import inference_base_command
+
+    inference_base_command(model, **kwargs)
```

Comment:
shouldn't this be returning the updated config? Or I guess it updates it in-place? I think what is happening here is a little opaque

Reply:
Decision: rename this function `apply_shared_inference_parameters()`, and call the interface `apply_shared_inference_parameters_interface`